netebpfext/net_ebpf_ext_bind.c

```diff
@@ -273,7 +273,19 @@ net_ebpf_ext_resource_allocation_classify(
     }
 
     attached_client = (net_ebpf_extension_hook_client_t*)filter_context->client_context;
-    ENTER_HOOK_CLIENT_RUNDOWN(attached_client);
+    if (!net_ebpf_extension_hook_client_enter_rundown(attached_client)) {
+        // In some stress-testing related edge cases (especially where we unload/reload this extension rapidly) we end
```

Comment:
This comment should be removed from here IMO. If you must, put it inside `net_ebpf_extension_hook_client_enter_rundown` .

Reply:
I waffled between the two for a bit, but thought it seemed to make more sense _here_ as this is where we take the specific _action_ based on the return value.  IMO, the `net_ebpf_extension_hook_client_enter_rundown` ought to just return the result of the operation ('mechanism') as opposed to the processing thereof ('policy') which is best determined by the caller.

Case in point is the call from the `net_ebpf_extension_wfp_filter_context_create` where we end up calling `fastfail` in response to the error result.  

To my mind, inability to acquire a rundown is serious enough an event that it deserves a comment _where-ever_ it occurs, hence IMO the repetition is justified.  Also, the comment may seem overly repetitive in _this_ PR, but in reality, the affected code file(s) lie in different contexts. 

Thoughts?

Reply:
I really don't like having multiple copies of something, be it comment or code. It will invariably end up getting out of synch. Maybe turn it into a macro?

Reply:
Removed the comment ( A macro did not really buy anything significant).